drivers/lora/lora_basics_modem/lbm_common.h

```diff
@@ -54,11 +54,8 @@ struct lbm_lora_data_common {
 			int16_t rssi_dbm;
 			int8_t snr_db;
 		} sync;
-		struct {
-			/* Async RX params */
-			lora_recv_cb rx_cb;
-			void *user_data;
-		} async;
+		/* Async RX params */
+		struct lora_recv_async_callbacks async;
```

Comment:
This should be part of "lora: single asynchronous RX parameter" (previous commit), right?